service/history/queueProcessor.go

```diff
@@ -283,7 +283,13 @@ ProcessRetryLoop:
 			if err != nil {
 				if err == ErrTaskRetry {
 					p.metricsClient.IncCounter(p.options.MetricScope, metrics.HistoryTaskStandbyRetryCounter)
-					<-notificationChan
+				DelayLoop:
```

Comment:
Looks like we will delay the task after processing for first time.  I thought the idea is to even delay on first processing.

Reply:
for transfer task, there is no timestamp associated with it.
we can tag the queue processor, however, that require more code.